our-brothers-web/src/app/components/header/header.component.scss

```diff
@@ -181,34 +143,80 @@
   }
 }
 
-@media only screen and (max-width: 1400px) {
-  .links-header {
-    .link-header {
-      &:not(:last-child) {
-        margin-left: 20px !important;
-      }
-    }
-  }
-  .logo-header {
-    background-repeat: no-repeat;
-    flex: 0 0 120px;
-    height: 50px;
-  }
-}
-
-@media only screen and (max-width: 700px) {
-  .header-out {
-    justify-content: center;
-  }
+@media only screen and (max-width: 1200px) {
   .header {
-    justify-content: center;
+    position: fixed;
+    top: 0;
+    right: 0;
+    width: 300px;
+    height: 100%;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: stretch;
+    padding: 1.5em;
+    box-shadow: -10px 0px 2px 0px rgba(0,160,153,0.75);
+
+    .close-drawer-menu {
+      display: block;
+      color: rgba(0,160,153);
+      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 10 10' enable-background='new 0 0 40 40'%3E%3Cline x1='0' y1='0' x2='10' y2='10' stroke='rgba(0,160,153)' stroke-width='1' stroke-linecap='round' stroke-miterlimit='10'%3E%3C/line%3E%3Cline x1='0' y1='10' x2='10' y2='0' stroke='rgba(0,160,153)' stroke-width='1' stroke-linecap='round' stroke-miterlimit='10'%3E%3C/line%3E%3C/svg%3E");
```

Comment:
Can we extract it to SVG file?

Reply:
we can but why?

Reply:
Better maintenance in the future. But feel free to resolve it as is

Reply:
I moved the class to syle.css and changed it to .site-close-icon